fast64_internal/oot/oot_scene_room.py

```diff
@@ -172,13 +172,24 @@ def drawExitProperty(layout, exitProp, index, headerIndex, objName):
 
 class OOTObjectProperty(bpy.types.PropertyGroup):
     expandTab: bpy.props.BoolProperty(name="Expand Tab")
-    objectID: bpy.props.EnumProperty(items=ootEnumObjectID, default="OBJECT_HUMAN")
-    objectIDCustom: bpy.props.StringProperty(default="OBJECT_HUMAN")
+    objectID: bpy.props.EnumProperty(items=ootData.objectData.ootEnumObjectID, default="obj_human")
+    objectIDLegacy: bpy.props.EnumProperty(items=ootData.objectData.ootEnumObjectIDLegacy, default="OBJECT_HUMAN")
+    objectIDCustom: bpy.props.StringProperty(default="OBJECT_CUSTOM")
+
+    @staticmethod
+    def upgrade_object(obj):
+        if obj.data is None:
+            if (obj.ootEmptyType == "Scene") or (obj.ootEmptyType == "Room"):
+                print(f"Processing '{obj.name}'...")
+                for child in obj.children:
+                    OOTObjectProperty.upgrade_object(child)
+            elif obj.fast64.oot.version < obj.fast64.oot.cur_version:
```

Comment:
so, `if (obj.ootEmptyType == "Scene") or (obj.ootEmptyType == "Room")` then `upgradeObjectInit` doesn't run? what (blender) objects does it actually run on?

also
```suggestion
            elif obj.fast64.oot.version < 1:
```
(for example changing from 1 to 2 wouldn't need upgradeObjectInit)